lib/python/gladevcp/hal_python.xml

```diff
@@ -63,7 +62,7 @@
         <glade-widget-class name="HAL_VBar" generic-name="hal_vbar" title="HAL VBar"/>
         <glade-widget-class name="Override" generic-name="override" title="Override Controls">
             <properties>
-                <property id="Adjustment" visible="false"/>
+                <property id="Adjustment" visible="False"/>
```

Comment:
I do not remember why I changed 'f' to 'F'. I really do not know if there is any difference.
I put it as before.